### PR TITLE
MAGECLOUD-3520 update screenshot, php lines, markdown

### PR DESCRIPTION
**Pull request by hguthrie**
This PR updates the inaccuracies related to multi-sites outlined in the MAGECLOUD-3520:

- update screenshot in Config guide
- PHP code snippet formatting fixes
- fix multi-site by differentiating between sites with shared domain and sites with unique domains

whatsnew
Update the [Set up multiple website or stores](https://devdocs.magento.com/guides/v2.3/cloud/project/project-multi-sites.html) topic in the Cloud guide. Now, this topic differentiates between sites with a shared domain and sites with unique domains.

### CODE REVIEW

File: guides/v2.2/cloud/project/project-start.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ File                      | Description
...
 `composer.json`           | Fetches the Magento Enterprise Edition and the necessary configuration scripts to prepare your application. See [Prepare your Magento install]({{ page.baseurl }}/cloud/setup/first-time-setup-import-prepare.html).
 `composer.lock`           | Stores version dependencies for every package.
-`magento-vars.php`        | A file used to define [multiple stores]({{ page.baseurl }}/cloud/project/project-multi-sites.html#modify-the-magento-varsphp-file) and sites using [Magento variables]({{ page.baseurl }}/config-guide/multi-site/ms_over.html).
-
+`magento-vars.php`        | A file used to define [multiple stores]({{ page.baseurl }}/cloud/project/project-multi-sites.html#modify-magento-variables) and sites using [Magento variables]({{ page.baseurl }}/config-guide/multi-site/ms_over.html).
 
+http://127.0.0.1:4000/guides/v2.3/cloud/project/project-multi-sites.html#modify-magento-variables
```
Comment (meker12) [inline guides/v2.2/cloud/project/project-start.md]:
```suggestion
```

File: guides/v2.2/cloud/project/project-multi-sites.md [hunk 2]
```diff
@@ -8,157 +8,245 @@ functional_areas:
...
+   "https://*.{default}/":
+       type: upstream
+       upstream: "mymagento:php"
+   ```
+
+1. Save your changes to the `routes.yaml` file.
+
+### Configure locations for shared domains {#locations}
+
+Where routes defines how the URLs are processed, the `web` property in the `.magento.app.yaml` file defines how your application is exposed to the web. Web _locations_ allows more granularity for incoming requests. For example, if your domain is `store.com`, you can use `/first` and `/second` for requests to two different stores that share that domain.
```
Comment (meker12) [inline guides/v2.2/cloud/project/project-multi-sites.md]: This overview mentions the `/first` and `/second` paths, but the example only shows the config for `/second`.

Comment (hguthrie) [reply guides/v2.2/cloud/project/project-multi-sites.md]: Yep. The first is the default.

File: guides/v2.2/cloud/project/project-start.md (hunk 1 above)
Comment (hguthrie) [reply guides/v2.2/cloud/project/project-start.md]: hahahahah, how did that get there? ! 😆

File: guides/v2.2/cloud/project/project-multi-sites.md [hunk 3]
```diff
@@ -8,157 +8,246 @@ functional_areas:
...
+1. Open the `.magento/routes.yaml` file in a text editor.
+
+1. Replace the contents with the following:
+
+   ```yaml
+   "http://{default}/":
+       type: upstream
+       upstream: "mymagento:http"
+
+   "https://{default}/":
```
Comment (NadiyaS) [inline guides/v2.2/cloud/project/project-multi-sites.md]: right now, routes with https protocol will be added by default if you have the same for http, so no needs to add them. Probably we need to add this info where we describe how to work with routes. And add link to routes description in this article too.

File: guides/v2.2/cloud/project/project-multi-sites.md [hunk 4]
```diff
@@ -8,157 +8,246 @@ functional_areas:
...
+   "https://<second-site>.{default}/":
+       type: upstream
+       upstream: "mymagento:https"
+   ```
+
+1. Save your changes to the `routes.yaml` file.
+
+### Configure locations for shared domains {#locations}
+
+Where routes defines how the URLs are processed, the `web` property in the `.magento.app.yaml` file defines how your application is exposed to the web. Web _locations_ allows more granularity for incoming requests. For example, if your domain is `store.com`, you can use `/first` (default site) and `/second` for requests to two different stores that share that domain.
```
Comment (NadiyaS) [inline guides/v2.2/cloud/project/project-multi-sites.md]: Is it typo with define(s)?

File: guides/v2.2/cloud/project/project-multi-sites.md [hunk 5]
```diff
@@ -8,157 +8,246 @@ functional_areas:
...
+
+```yaml
+web:
+    locations:
+        "/": &app
```
Comment (NadiyaS) [inline guides/v2.2/cloud/project/project-multi-sites.md]: I do not know how, but we need to mark that we added alias `app` to "/" and use it for `/<second-site>` for avoiding duplication

File: guides/v2.2/cloud/project/project-multi-sites.md [hunk 6]
```diff
@@ -8,157 +8,246 @@ functional_areas:
...
+            scripts: false
+            passthru: "/front-static.php"
+            rules:
+                ^/static/version\d+/(?<resource>.*)$:
+                    passthru: "/static/$resource"
+        "/<second-site>": *app
+          ...
+```
+
+You can also add the directory to the `static` location rules. For example:
```
Comment (NadiyaS) [inline guides/v2.2/cloud/project/project-multi-sites.md]: if not to add directory for static, we need to change URLs in system configurations (Magento admin > Stores > Configurations > General > Web > (Secure) Base URL for Static View Files

File: guides/v2.2/cloud/project/project-multi-sites.md [hunk 7]
```diff
@@ -8,157 +8,246 @@ functional_areas:
...
+   }
+   ```
+   {:.no-copy}
+
+1. Change the following line:
+
+   From:
+
+   ```php?start_inline=1
+   return strpos(str_replace('---', '.', $_SERVER['HTTP_HOST']), $host) === 0;
```
Comment (NadiyaS) [inline guides/v2.2/cloud/project/project-multi-sites.md]: I guess we need to change it in the file itself. Do not see reasons to keep example with `---`

File: guides/v2.2/cloud/project/project-multi-sites.md [hunk 8]
```diff
@@ -8,157 +8,246 @@ functional_areas:
...
-    ```bash
-    git add -A && git commit -m "Implement multiple sites" && git push origin <branch name>
-    ```
+1. Wait for deployment to complete.
 
-1.  Wait for deployment to complete.
+1. After deployment, open your site base URL in a web browser.
 
-1.  After deployment, open your site base URL in a web browser.
+   Use the format: `http://<magento-run-code>.<site-URL>`
```
Comment (NadiyaS) [inline guides/v2.2/cloud/project/project-multi-sites.md]: is this section only for stores with subdomains? if no, probably need to rephrase like "open store  URL based on configuration you made, for example for subdomains use format..."

File: guides/v2.2/cloud/project/project-multi-sites.md (hunk 4 above)
Comment (hguthrie) [reply guides/v2.2/cloud/project/project-multi-sites.md]: Well, technically it is referencing the previous section about the routes file. I'll clear it up.

**Comment by hguthrie** [comment]
Yay! Major technical updates completed! @meker12 You are welcome to review again, if you like. Otherwise, this is ready to go!

**Comment by hguthrie** [comment]
running tests

File: guides/v2.2/cloud/project/project-multi-sites.md [hunk 9]
```diff
@@ -8,157 +8,310 @@ functional_areas:
...
+       upstream: "mymagento:http"
+   ```
+
+1. Save your changes to the `routes.yaml` file.
+
+1. Continue to the [_Set up websites, stores, and store views_ section](#set-stores).
+
+### Configure locations for shared domains {#locations}
+
+Where the routes configuration defines how the URLs are processed, the `web` property in the `.magento.app.yaml` file defines how your application is exposed to the web. Web _locations_ allows more granularity for incoming requests. For example, if your domain is `store.com`, you can use `/first` (default site) and `/second` for requests to two different stores that share the same domain.
```
Comment (meker12) [inline guides/v2.2/cloud/project/project-multi-sites.md]:
```suggestion
Where the routes configuration defines how the URLs are processed, the `web` property in the `.magento.app.yaml` file defines how your application is exposed to the web. Web _locations_ allow more granularity for incoming requests. For example, if your domain is `store.com`, you can use `/first` (default site) and `/second` for requests to two different stores that share the same domain.
```

Comment (hguthrie) [reply guides/v2.2/cloud/project/project-multi-sites.md]: I saw this after the test run, but yes, this is tricky. Locations is a web property category. The brain can get confused here. I'd like to find a way to get rid of Allow. I don't like using that word, in fact. Maybe provides.... ugh, staring too long...

**Pull request merged**